MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -388,6 +388,35 @@ public void UpdateServerTimeOffset(long currentServerTime)
 			AnimationManager.UpdateServerTimeOffset(currentServerTime);
 		}
 
+		public void DeclarePreallocatedActors(GameObject[] objects, string guidSeed)
```

Comment:
We need to protect against this getting called multiple times, or allow it to be called multiple times safely, potentially with the same guidSeed.

Reply:
I like the idea of making sure that seeds aren't reused. Technically it's fine if it gets called multiple times, it's just less efficient because it sends multiple object-spawned messages.